project-keeper/src/main/java/com/exasol/projectkeeper/validators/pom/plugin/SimplePluginTemplateGenerator.java

```diff
@@ -0,0 +1,37 @@
+package com.exasol.projectkeeper.validators.pom.plugin;
+
+import java.util.Collection;
+import java.util.Optional;
+
+import org.w3c.dom.Node;
+
+import com.exasol.projectkeeper.ProjectKeeperModule;
+
+/**
+ * Validator for maven-deploy-plugin.
+ */
+public class SimplePluginTemplateGenerator implements PluginTemplateGenerator {
+    private final String templateResource;
+    private final ProjectKeeperModule module;
+
+    /**
+     * Create a new instance of {@link SimplePluginTemplateGenerator}.
+     * 
+     * @param templateResource path to the template
+     * @param module           module to use this plugin for
+     */
+    public SimplePluginTemplateGenerator(final String templateResource, final ProjectKeeperModule module) {
```

Comment:
Is it possible to make this package-private? Then we can avoid the annoying standard javadoc comment.
Or you use Lombok magic to create the constructor ;)